src/gl/canvas2dRe.re

```diff
@@ -1,4 +1,10 @@
+/* don't refmt this file until refmt stops stripping bs.as. i don't know the issue number,
+  but have been told it's fixed-but-unreleased */
```

Comment:
BuckleScript uses refmt internally to convert from reason to ocaml, so it doesn't really matter whether or not you do it manually. The attributes will continue to be dropped until the fix has been released. But we should leave a comment here to remind ourselves to check that it's actually fixed once it does get released.

The issue is https://github.com/facebook/reason/issues/821